src/Admin/Product/Attributes/AttributesForm.php

```diff
@@ -132,6 +133,13 @@ public static function init_input( InputInterface $input, AttributeInterface $at
 				$new_input->set_label( $input->get_label() )
 					->set_description( $input->get_description() );
 
+				// Copy values from GTIN
```

Comment:
This comment didn't help clarify much. Can we elaborate a bit more here:

```suggestion
				// When GTIN uses the SelectWithTextInput field, copy the readonly/hidden attributes from the GTINInput field.
```